### PR TITLE
Jupytext

### PR DESCRIPTION
**Pull request by wd15**
Use jupytext for notebooks in results/ directory. Using markdown format as the scripts don't work outside of a notebook context.

### CODE REVIEW

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1494.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1494.surge.sh

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1494.surge.sh

**Comment by wd15** [comment]
@tkphd, this is ready for review

File: results/benchmark1b.1.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.2-dev
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python papermill={"duration": 0.026347, "end_time": "2023-03-07T17:19:41.523290", "exception": false, "start_time": "2023-03-07T17:19:41.496943", "status": "completed"} tags=["parameters"]
```
Comment (wd15) [inline results/benchmark1b.1.md:16]: @tkphd, this papermill metadata changes everytime the notebooks are rebuilt, which is very  annoying. Unfortunately, I can't switch off Jupytext saving the metadata as we need the parameters tag for the parameter injection. Very annoying.

Comment (wd15) [reply results/benchmark1b.1.md:16]: Jupytext has thankfully removed all of the Jupyter metadata with times and unique IDs that keeps changing. Unfortunately, papermill is doing the same thing.

File: .travis.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ install:
 before_script:
   - nix-shell _apps/data/shell.nix --pure --command "echo 'run nix-shell and exit'"
 script:
+  - cd _data/python-pfhub
+  - nix-shell --command "cd ../../results && make notebooks"
+  - cd ../..
   - nix-shell --command "
```
Comment (tkphd) [inline .travis.yml:26]:
```suggestion
  - nix-shell --command "cd results && make notebooks"
  - nix-shell --command "
```

File: results/benchmark1b.1.md (hunk 1 above)
Comment (tkphd) [reply results/benchmark1b.1.md:16]: What if we load [`papermill-jupytext`](https://pypi.org/project/papermill-jupytext/) instead of vanilla `papermill`?

**Review by tkphd — CHANGES_REQUESTED** [review]
I :sparkling_heart: removing the crufty `.raw` and `.ipynb` files for each benchmark! A couple minor comments on workflow and papermill-jupytext compatibility, but this looks great!

File: results/benchmark1b.1.md (hunk 1 above)
Comment (wd15) [reply results/benchmark1b.1.md:16]: `papermill-jupytext` doesn't inspire me as it has [18 commits and hasn't been touched for four years](https://github.com/mwouts/papermill_jupytext).

File: .travis.yml (hunk 2 above)
Comment (wd15) [reply .travis.yml:26]: The [shell.nix](https://github.com/usnistgov/pfhub/blob/master/_data/python-pfhub/shell.nix) file needs to be launched from the directory with the python package in it if it is used as a development package. This is the shell.nix file in `_data/python-pfhub`. Using as a development package means that edited python files can be immediately be used in the notebooks without rerunning the nix install. This is a weird interaction with setuptools and nix installing from the nix-shell. If pfhub is listed as a package in the nix environment than this wouldn't be the case, but then it isn't a dev package and all it's files are stored in the nix-store and need to be updated everytime I make a change the python-pfhub and then run a notebook.

**Comment by wd15** [comment]
@tkphd, please check if I've addressed your concerns.

**Review by tkphd — APPROVED** [review]
LGTM!

**Pull request merged**